src/stories/2-InputHash.stories.js

```diff
@@ -8,5 +8,14 @@ export default {
   component: InputHash
 }
 
-export const InputHashCreator = () => <InputHash handleOnChange={action('Result')} />
-export const InputHashValidator = () => <InputHash useHashValidator handleOnChange={action('Result')} />
\ No newline at end of file
+export const InputHashCreator = () => (
+  <InputHash handleOnChange={action('Result')} />
+)
+export const InputHashValidator = () => (
+  <InputHash
+    useHashValidator
+    handleOnChange={action('Result')}
+    messageSuccess='Longitud de hash correcta'
```

Comment:
should we define this string in props?

Reply:
it's already define in inputHash props, in any case, it is optional